src/sentry/api/endpoints/release_deploys.py

```diff
@@ -119,13 +119,14 @@ def post(self, request, organization, version):
                 last_deploy_id=deploy.id,
             )
 
-            ReleaseProjectEnvironment.objects.filter(
-                release=release,
-                environment=env,
-                project__in=projects,
-            ).update(
-                last_deploy_id=deploy.id,
-            )
+            for project in projects:
+                ReleaseProjectEnvironment.get_or_create(
```

Comment:
i think you wanna use `create_or_update` (which is something we've added since that functionality didn't exist in the django version we're using) https://github.com/getsentry/sentry/blob/master/src/sentry/db/models/query.py#L57